umbraco/Umbraco-CMS
# New backoffice: Relation controller

# Notes
- Added `RelationViewModel` instead of `RelationDisplay`
- Added new `RelationViewModelsMapDefinition`
- Added new `IRelationViewModelFactory` to create the RelationViewModels instead of mappers using services
- Ported `GetById` & `GetByChildId` to new individual controllers with new endpoints

# How to test
- Go to the media section and upload some images
- Create a document type
- Add a Image Picker to the document type
- Create some content with your document type, and add the images
- Go to the picked image, it should now under "info" have relations

Call the Get endpoint and assert you can see the relation you've just created
(hint: you can find the id in the database, under the umbracoRelation table)
https://localhost:44331/umbraco/api/v1.0/relation/1

- You should get a response with the relataion and it should have a childId property
- Use that child id to call the other endpoint:

https://localhost:44331/umbraco/api/v1.0/relation/childRelations/1058?take=100
- Assert you get the relation in the response
